.vortex/installer/src/Prompts/Handlers/CustomModules.php

```
@@ -130,16 +129,7 @@ public function process(): void {
     if (!in_array(self::BASE, $selected)) {
       File::removeTokenAsync('CUSTOM_MODULE_BASE');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\DrevOps\VortexInstaller\Utils\File' in method 'process'. (undefined)

(StaticAccess)